src/components/Timetable/UploadPage.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  Button,
+  Stack,
+  Typography,
+  TextField,
+  Link,
+  InputAdornment,
+  IconButton,
+} from '@mui/material';
+import SendIcon from '@mui/icons-material/Send';
+import styles from './UploadPage.module.css';
+
+/**
+ * This component can be used to display the upload page for the app after the user has logged in.
+ * Users can input their timetable details into this page. Currenly the page is not functional.
+ *
+ */
+export default function UploadPage() {
+  const [calendarURL, setcalendarURL] = React.useState('');
+  const handleChange = (event) => {
+    setcalendarURL(event.target.value);
+  };
+
+  return (
+    <Stack direction="column" spacing={3} className={styles.title}>
+      <Button variant="contained">Upload Timetable</Button>
+      <Typography>OR</Typography>
+      <TextField
+        id="outlined-input"
+        aria-label="Enter URL"
+        label="Enter your calendar URL"
+        value={calendarURL}
+        onChange={handleChange}
+        InputProps={{
+          endAdornment: (
+            <InputAdornment position="end">
+              <IconButton edge="end" color="primary" aria-label="submit">
```

Comment:
would be nice if there is a form for this submit or the button calls a function.

e.g sendTimetableICS(link) and this is pass through the props.

Reply:
Good idea, I'll pass in a function as a prop.

Reply:
@scorpionknifes Have added it now